_authors/smortex.markdown

```diff
@@ -0,0 +1,7 @@
+---
+short_name: smortex
+name: Romain Tartière
+photo: '/assets/media/authors/smortex.jpg'
+github: smortex
+---
+**Romain Tartière** is a Hacker, FLOSS Mercenary, FreeBSD developer (romain@), BOFH, love stuff from the 70's (e.g. UNIX, Japanese hard-rock) and Ruby.
```

Comment:
Thanks for taking care of my "bio" (I am terrified by this kind of things :upside_down_face:)!  Not sure about this, I feel like what I wrote at the 1st person (I am…) and was copied needs an extra 's' in this form:

```suggestion
**Romain Tartière** is a Hacker, FLOSS Mercenary, FreeBSD developer (romain@), BOFH; he loves stuff from the 70's (e.g. UNIX, Japanese hard-rock) and Ruby.
```

And maybe repeating `a` is better?  Maybe a native English speaker can help :grinning: 

```suggestion
**Romain Tartière** is a Hacker, a FLOSS Mercenary, a FreeBSD developer (romain@), a BOFH. He loves stuff from the 70's (e.g. UNIX, Japanese hard-rock) and Ruby.
```

Thanks!